File: test/fixtures/output/ruby/native/unparseable-query.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har?&&a=b&&")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body

File: test/fixtures/output/javascript/jquery/unparseable-query.js
const settings = {
  "async": true,
  "crossDomain": true,
  "url": "http://mockbin.com/har?&&a=b&&",
  "method": "GET",
  "headers": {}
};

$.ajax(settings).done(function (response) {
  console.log(response);
});